mix.exs

```diff
@@ -23,7 +29,45 @@ defmodule PgQuery.MixProject do
   defp deps do
     [
       {:elixir_make, "~> 0.4", runtime: false},
-      {:protox, "~> 1.7"}
+      {:protox, "~> 1.7"},
+      {:ex_doc, ">= 0.0.0", only: :dev, runtime: false}
+    ]
+  end
+
+  defp package do
+    # Exclude the test files and other bits and bobs to sneak us in under the
+    # 8MB hex.pm package size limit...
+    c_src = ~w(
+      c_src/libpg_query_ex.c
+      c_src/libpg_query/LICENSE
+      c_src/libpg_query/Makefile
+      c_src/libpg_query/README.md
+      c_src/libpg_query/pg_query.h
+      c_src/libpg_query/patches
+      c_src/libpg_query/protobuf
+      c_src/libpg_query/src
+      c_src/libpg_query/vendor
+    )
+
+    [
+      files: ~w(lib .formatter.exs mix.exs README.* LICENSE Makefile) ++ c_src,
+      licenses: ["Apache-2.0"],
+      links: %{
+        "libpg_query" => "https://github.com/pganalyze/libpg_query",
+        "Github" => "https://github.com/electric-sql/pg_query_ex",
+        "Electric SQL" => "https://electric-sql.com"
+      }
+    ]
+  end
+
+  defp description do
+    "An Elixir interface to the lib_pg_query postgres query parser. Converts a valid PostgreSQL query into a Protobuf based AST."
```

Comment:
The spelling for `lib_pg_query` here different from the one under the `links` key above.

Reply:
Good spot! Thanks.